psi4/driver/driver.py

```diff
@@ -1738,7 +1738,9 @@ def molden(wfn, filename=None, density_a=None, density_b=None, dovirtual=None):
     >>> # [4] This WILL work, note the transformation of Da (SO->MO)
     >>> E, wfn = properties('ccsd', properties=['dipole'], return_wfn=True)
     >>> Da_so = wfn.Da()
-    >>> Da_mo = core.triplet(wfn.Ca(), Da_so, wfn.Ca(), True, False, False)
+    >>> SCa = Matrix("Overlap times Ca", nmopi, nmopi)
+    >>> SCa.gemm(False, False, 1.0, wfn.S(), wfn.Ca(), 0.0)
+    >>> Da_mo = core.triplet(SCa, Da_so, SCa, True, False, False)
```

Comment:
`SCa` would be `nsopi x nmopi`. In any case,  using `SCa = core.doublet(wfn.S(), wfn.Ca(), False, False)` avoids the issue all together.

Reply:
Thanks. Good idea and I also forgot to define `nmopi`. All should be fine now.